packages/components/src/components/audio/audio.ts

```diff
@@ -399,52 +387,42 @@ export default class SdAudio extends SolidElement {
       </button>
 
       <div class="flex items-center justify-self-end">
-        ${this.hasTranscript
-          ? html` <sd-icon
-              name="transcript"
-              library="system"
+        ${this.hasSlotController.test('transcript')
+          ? html`<button
               class=${cx(
                 'mr-6 w-6 h-6 hover:cursor-pointer hover:text-primary-500',
                 this.inverted
                   ? 'text-white focus-visible:focus-outline-inverted'
                   : 'text-primary focus-visible:focus-outline'
               )}
-              tabindex="0"
               @click=${this.showTranscript}
               @keydown=${this.showTranscriptKeydown}
-            ></sd-icon>`
+              tab-index="0"
+            >
+              <sd-icon
+                class="w-6 h-6"
+                name="transcript"
+                library="system"
+                label=${this.isMuted ? this.localize.term('unmute') : this.localize.term('mute')}
+              ></sd-icon>
+            </button>`
           : null}
 
-        <div class="audio-player__volume-container flex items-center" part="volume">
-          <sd-icon
-            class=${cx(
-              'audio-player__volume-icon w-6 h-6 hover:cursor-pointer focus:mr-2 hover:text-primary-500',
-              this.inverted
-                ? 'text-white focus-visible:focus-outline-inverted'
-                : 'text-primary focus-visible:focus-outline'
-            )}
-            name=${this.isMuted ? 'mute' : 'volume'}
-            library="system"
-            aria-label=${this.localize.term('mute')}
-            tabindex="-1"
-            @click=${this.muteAudio}
-            @keydown=${this.muteAudioKeydown}
-          ></sd-icon>
-          <!-- TODO: replace with sd-range once it's implemented -->
-          <input
-            class=${cx(
-              'audio-player__volume-slider appearance-none cursor-pointer outline-none h-1 w-0 overflow-hidden',
-              this.inverted ? 'bg-white' : 'bg-primary'
-            )}
-            type="range"
-            max="100"
-            value=${this.volume * 100}
-            tabindex="0"
-            label="Volume"
-            @click=${this.handleVolumeChange}
-            @keydown=${this.handleVolumeChangeKeydown}
-          />
-        </div>
+        <button
+          class=${cx(
+            'w-6 h-6 hover:cursor-pointer hover:text-primary-500',
+            this.inverted
+              ? 'text-white focus-visible:focus-outline-inverted'
+              : 'text-primary focus-visible:focus-outline'
+          )}
+          part="volume"
+          aria-label=${this.localize.term('mute')}
+          tabindex="0"
+          @click=${this.toggleMute}
+          @keydown=${this.toggleMuteKeydown}
+        >
+          <sd-icon class="w-6 h-6" name=${this.isMuted ? 'mute' : 'volume'} library="system"></sd-icon>
```

Comment:
what was the latest agreement on the slider? It is completely gone in the feature branch deployments.

Reply:
The volume slider was presenting some ux and implementation issues so we decided to scrap it for now and go with simpler version. If we see it is something our users need, we will find a way to include it in a future iteration.